en/physics-2d/physics-2d-system.md

```diff
@@ -0,0 +1,158 @@
+    EPhysics2DDrawFlags.Pair |
+    EPhysics2DDrawFlags.CenterOfMass |
+    EPhysics2DDrawFlags.Joint |
+    EPhysics2DDrawFlags.Shape;
+```
+
+Set the drawing flag to **EPhysics2DDrawFlags.None** to disable drawing.
+
+```ts
+PhysicsSystem2D.instance.debugDrawFlags = EPhysics2DDrawFlags.None;
+```
+
+### Conversion the physics units to the world coordinate system units
+
+General Physical Modules(Box2d) uses **Metre-Kilogramme-Second (MKS)** unit system, it has the best performance operating under such a unit system. But we use the **world coordinate system units** (short for world units) as the unit of length in 2D games, so we need a ratio to convert the physics units to the world units.
+
+In general, we set this ratio to **32**, which can be obtained by `cc.PhysicsManager.PTM_RATIO`, and this value is read-only. Usually the user does not need to care about this value, the physics system will automatically convert the physics units and world units to each other. User can use the familiar world units for all the calculations.
+
+### Set physics gravity
+
+Gravity is a very important thing in physics operation, and most physics games use the gravity as a important feature.
+
+The default gravity is `(0, -320)` world units per second^2, according to the conversion rules described above, that's `(0, -10)` m/s^2 in physics unit.
+
+If you want the gravity to be 0, you can set like this:
+
+```ts
+PhysicsSystem2D.instance.gravity = cc.v2();
+```
+
+If you wish to change the acceleration of gravity to something else, such as a `20 m/s` acceleration landing, then you can set the acceleration to.
+
+```ts
+PhysicsSystem2D.instance.gravity = cc.v2(0, -20 * PHYSICS_2D_PTM_RATIO);
+```
+
+### Set physics timestep
+
+The Physics System updates the physical world according to a fixed timestep, the default timestep is `1/60`. But some games may not want to follow such a high frequency to update the physical world, after all, this operation is more time consuming, then you can reduce the timestep to achieve this effect.
+
+```ts
+const system = PhysicsSystem2D.instance;
+
+// Physics timestep, default fixedTimeStep is 1/60
+system.fixedTimeStep = 1/30;
+
+// The number of iterations per update of the Physics System processing speed is 10 by default
+system.velocityIterations = 8;
+
+// The number of iterations per update of the Physics processing location is 10 by default
+system.positionIterations = 8;
+```
+
+**Notice**: Reducing the fixed timestep and the number of iterations for each property will reduce the physics detection frequency. Therefore, it is more likely to occur rigid body penetration, which needs to be taken into account when using.
+
+## Query physics object
+
+Often you may want to know which physics objects are in a given scene. For example, if a bomb explodes, objects in the range will be damaged; or in a strategy game, you may want to let the user drag to move a unit from a certain range.
```

Comment:
Often you may want to know which physics objects are in a given scene.
-->
Often, knowing which physics objects are in a given scene is beneficial.